CHANGELOG.md

```diff
@@ -29,6 +29,19 @@
 - [#1207](https://github.com/oauth2-proxy/oauth2-proxy/pull/1207) Fix URI fragment handling on sign-in page, regression introduced in 7.1.0 (@tarvip)
 - [#1244](https://github.com/oauth2-proxy/oauth2-proxy/pull/1244) Update Alpine image version to 3.14 (@ahovgaard)
 
+# V7.1.4
```

Comment:
You should remove this section. The maintainers take care of this when we make a release.

You can add your CHANGELOG entry up above in the prerelease area.

Reply:
Done.